code/datums/brain_damage/phobia.dm

```diff
@@ -107,10 +107,14 @@
 	if(HAS_TRAIT(owner, TRAIT_FEARLESS))
 		return
 	if(trigger_regex.Find(speech_args[SPEECH_MESSAGE]) != 0)
+		if(speech_args[SPEECH_FORCED])
+			// If we've been forced to speak a phobia word, allow it to go through - then freak out!
+			addtimer(CALLBACK(src, PROC_REF(freak_out), null, trigger_regex.group[2], TRUE), 1 SECONDS)
+			return
 		to_chat(owner, span_warning("You can't bring yourself to say the word \"[span_phobia("[trigger_regex.group[2]]")]\"!"))
 		speech_args[SPEECH_MESSAGE] = ""
 
-/datum/brain_trauma/mild/phobia/proc/freak_out(atom/reason, trigger_word)
+/datum/brain_trauma/mild/phobia/proc/freak_out(atom/reason, trigger_word, said_not_heard = FALSE)
 	COOLDOWN_START(src, scare_cooldown, 12 SECONDS)
```

Comment:
We're going in the wrong direction with this proc specifically, since it already has issues with the trigger_word. Could you:
- Move the reaction part of freak_out into its own proc
- Split freak_out into two procs, one handling atom reactions and one handling verbal reactions
- Both freak_out types call the reaction proc after

Will avoid having to provide a null as an argument on every freak_out call involving words.